integrationtesting/opendcs-tests/src/test/java/org/opendcs/fixtures/configurations/cwms/CwmsOracleConfiguration.java

```diff
@@ -32,9 +38,9 @@ public class CwmsOracleConfiguration implements Configuration
 {
     private static Logger log = Logger.getLogger(OpenDCSPGConfiguration.class.getName());
 
-    private static final String CWMS_ORACLE_IMAGE = System.getProperty("opendcs.cwms.oracle.image","docker pull gvenzl/oracle-xe:21.3.0-faststart");
+    private static final String CWMS_ORACLE_IMAGE = System.getProperty("opendcs.cwms.oracle.image","registry-public.hecdev.net/cwms/database-ready-ora-23.5:latest-dev");
```

Comment:
Is this a 'free' oracle here?

Reply:
Yes, the CWMS ready database is based on the oracle free images.